master/buildbot/test/unit/test_process_build.py

```diff
@@ -157,8 +157,8 @@ def testRunSuccessfulBuild(self):
         b.startBuild(FakeBuildStatus(), None, self.workerforbuilder)
 
         self.assertEqual(b.results, SUCCESS)
-        self.assert_(('startStep', (self.workerforbuilder.worker.conn,), {})
-                     in step.method_calls)
+        self.assertIn(('startStep', (self.workerforbuilder.worker.conn,), {}),
```

Comment:
This is a good candidates for refactor in TestCase base classe (we use a tiny abstract class between unittest.TestCase and our tests, so we can implement our how missing assert* methods)...

  self.assertMethodCalledWith(step, ('startStep', (self.workerforbuilder.worker.conn,), {}),